src/client_side_request.cc

```diff
@@ -1269,7 +1270,7 @@ ClientRequestContext::clientRedirectDone(const Helper::Reply &reply)
             // prevent broken helpers causing too much damage. If old URL == new URL skip the re-write.
             if (urlNote != NULL && strcmp(urlNote, http->uri)) {
                 AnyP::Uri tmpUrl;
-                if (tmpUrl.parse(old_request->method, urlNote)) {
+                if (tmpUrl.parse(old_request->method, SBuf(urlNote))) {
```

Comment:
```suggestion
                if (tmpUrl.parse(old_request->method, SBuf(urlNote))) { // TODO: Use SBuf from reply.notes
```

I do not insist on this change.

Reply:
This is more a TODO about changing the Notes::findFirst API to return an SBuf. When that change is made this user of it will naturally have to be updated. So not adding this here.